frontend/src/components/DashboardView/DashboardLegends.tsx

```diff
@@ -8,34 +8,48 @@ import type { ExportConfig } from './DashboardContent';
 
 interface DashboardLegendsProps {
   exportConfig?: ExportConfig;
+  legendVisible?: boolean; // from map state
+  legendPosition?: 'left' | 'right'; // from map state
 }
 
-const DashboardLegends = memo(({ exportConfig }: DashboardLegendsProps) => {
-  const classes = useStyles();
-  const mode = useSelector(dashboardModeSelector);
-  // Use export config if available, otherwise show normally
-  const isVisible = exportConfig?.toggles?.legendVisibility ?? true;
-  const position = exportConfig?.legendPosition ?? 1; // default right (1)
-  const scale = exportConfig?.legendScale ?? 0; // default 100% (0 means no reduction)
+const DashboardLegends = memo(
+  ({ exportConfig, legendVisible, legendPosition }: DashboardLegendsProps) => {
+    const classes = useStyles();
+    const mode = useSelector(dashboardModeSelector);
 
-  if (!isVisible) {
-    return null;
-  }
+    // Priority: exportConfig takes precedence if present, otherwise use map state values
+    const isVisible =
```

Comment:
If `exportConfig` takes precedence, is there any expectation that modifying legend visibility in export view should impact the “normal” dashboard view? Or is the idea that you can just override legend visibility _within_ the export view?